src/backend/address_space_core.h

```diff
@@ -84,7 +85,7 @@ namespace snmalloc
         // external_pointer, for example) will not attempt to follow this
         // "Metaslab" pointer.
         MetaEntry t(reinterpret_cast<Metaslab*>(next.unsafe_ptr()), nullptr, 0);
-        pagemap.set(address_cast(base), t);
+        pagemap.set_meta_data(address_cast(base), 1, t);
```

Comment:
`set_metaentry` might be a better name for this.  Metadata is a very general term.